api-client/src/main/java/com/takipi/api/client/request/event/EventsRequest.java

```diff
@@ -36,13 +38,17 @@ public String urlPath() {
 	protected int paramsCount() {
 		// One slot for the stacktace flag.
 		//
-		return super.paramsCount() + 1;
+		return super.paramsCount() + 1 + (this.breakdown ? 1 : 0);
 	}
 
 	@Override
 	protected int fillParams(String[] params, int startIndex) throws UnsupportedEncodingException {
 		int index = super.fillParams(params, startIndex);
-
+		
+		if (this.breakdown) {
```

Comment:
1. Order of adding to the query should match the order of members/c-tor for coherency.
2. No need for this if, as we explicitly want to serialize something, and not rely on default backend value for this param.